Subject: [PATCH] Merge branch 'develop' of github.com:kirillskyeng/pytest_proj into develop
---
Index: tests/test_arrs.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/tests/test_arrs.py b/tests/test_arrs.py
--- a/tests/test_arrs.py	(revision 4c4cdcd29e77f04b66d368901207a79f5fb9cc22)
+++ b/tests/test_arrs.py	(date 1686126008048)
@@ -1,4 +1,4 @@
-from utils import arrs
+from tests.utils import arrs
 
 
 def test_get():
diff --git a/utils/__init__.py b/tests/utils/__init__.py
rename from utils/__init__.py
rename to tests/utils/__init__.py
diff --git a/utils/arrs.py b/tests/utils/arrs.py
rename from utils/arrs.py
rename to tests/utils/arrs.py
